### ISSUE TITLE
Replace or remove Get involved on front page

### ISSUE THREAD

**Issue by pmkovar** [redhat-openstack/website#742]
See https://github.com/redhat-openstack/website/pull/741 for details.

**Comment by rbowen** [comment]
#746 is a first cut at this. However, now we've got duplication between the "join the conversation" bit and the "what's happening" section at the bottom.

**Comment by pmkovar** [comment]
@rbowen  I really like the Twitter feed but... Just thinking aloud here, what if we replaced **Get involved** (or another block) with graphics, or a nice diagram that would sum up the architecture? There's too much text on the front page already.

**Comment by pmkovar** [comment]
It would make more sense to place the architecture diagram next to **What is RDO?**, though.

**Comment by rbowen** [comment]
I'd like to see what you have in mind.

The Twitter block kind of has to go on the second row, because it is, by design, much wider than what we need in the top row. But everything is up for discussion. Let's see what you have in mind.

**Comment by pmkovar** [comment]
Good point about the Twitter block.

By diagram I meant something like https://www.packtpub.com/sites/default/files/Article-Images/B04000_12_08.png but for RDO. Some others I could find are too complex for a front page (https://hguemar.fedorapeople.org/slides/rdo-packaging-kilo/img/openstack-full-architecture.png).

More graphics could improve the look & feel I think. Look, for example, at the Ansible front page: https://www.ansible.com/.

I also like the stats there. Would it make sense, for instance, to list a number of packages available in RDO repos? Maybe something similar to https://dashboards.rdoproject.org/rdo-dev? < /brainstorm>

**Comment by rbowen** [comment]
Yeah, I've been trying to figure out how to incorporate the dashboard into the front page for some time, but I don't know how to do that.
